.github/workflows/ci.yml

```diff
@@ -153,6 +153,9 @@ jobs:
       - name: 🔎 Lint
         run: yarn lint
 
+      - name: 🌡 Test Types
+        run: yarn test:types --target '4.6,5.0,latest' # commas has to be escaped in PowerShell
```

Comment:
Can’t find what is the latest TypeScript version supported by RedWood. Trying out 4.6.

Reply:
5.2.2 :) - we lock our versions, so if possible it would be helpful to just use the installed version rather than specify the target on CI

Reply:
Hm.. 5.2.2 is the version for building (or type checking code base), but what is the lowest on consumer side? Public types can be imported by users and their projects are not required to use only 5.2.2, or?

'--target' is the range of versions in which the typings can be consumed. This is what I try to say (;

Reply:
Ah gotcha - at the moment we lock users to the same versions as the framework - we have them as dependencies of the FW. This is true for all other dependencies - so the only way they can change versions is using resolutions. 

I think just checking the current version would be ok :)

Reply:
Sounds good to me. Thanks for clarification.